src/signer.ts

```diff
@@ -140,6 +147,7 @@ export class URLSigner {
     const config: GetSignedUrlConfigInternal = Object.assign({}, cfg, {
       method,
       expiration: expiresInSeconds,
+      usableFrom: new Date(1000 * usableFromInSeconds),
```

Comment:
Nit: not sure I understand `1000 * ` in this case, is it needed? (Comparing to expiration and it's not used in this way.